CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v3.0.4
```

Comment:
I realize this is probably coming from the cherry-pick, but this should be a `# Unreleased` header for the change commit, and then there should be a later commit that is the version bump on its on.